src/rgw/rgw_sal_motr.cc

```diff
@@ -3227,10 +3250,11 @@ int MotrMultipartWriter::complete(size_t accounted_size, const std::string& etag
   p.append(buf);
   string tenant_bkt_name = get_bucket_name(head_obj->get_bucket()->get_tenant(), head_obj->get_bucket()->get_name());
   string obj_part_iname = "motr.rgw.object." + tenant_bkt_name + "." +
-	                  head_obj->get_key().to_str() + ".parts";
-  ldpp_dout(dpp, 20) << "MotrMultipartWriter::complete(): object part index = " << obj_part_iname << dendl;
+                    head_obj->get_key().to_str() + ".parts";
+  ldpp_dout(dpp, 20) << __func__ << ": object part index = " << obj_part_iname << dendl;
   rc = store->do_idx_op_by_name(obj_part_iname, M0_IC_PUT, p, bl);
   if (rc < 0) {
+    ldpp_dout(dpp, 0) << __func__ << ": failed to add part obj in part index, rc=" << rc << dendl;
```

Comment:
At some instances we have "rc=" (without spaces) and at some other instances we have "rc = " (with spaces) and I think the later looks better. Can you please make all uniform (either with or without spaces)?